cmd/node/config/config.toml

```diff
@@ -147,6 +147,13 @@
         MaxBatchSize = 1
         MaxOpenFiles = 10
 
+[TrieSnapshotDB]
+     FilePath = "TrieSnapshot"
+     Type = "LvlDBSerial"
+     BatchDelaySeconds = 2
```

Comment:
should not be more than a round ? same question for evistion waitinglist

Reply:
Updated to 15. Updated the eviction waiting list to BatchDelaySeconds = 5. The waiting list needs to be stored in a database as soon as possible, so the timeframe of a round should be ok.